ml-conversational-analytic-tool/test_commentAnalysis.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from commentAnalysis import CommentAnalyzer
+
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    
+    def test_CommentAnalyzer_init(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        self.assertEqual(analyzer.word_count, {'test': 0})
+        """ negative test """
+        self.assertNotEqual(analyzer.word_count, {'test': 1})
+
+    def test_analyze_comments(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        result = analyzer.analyzeComment("This is a test comment")
+        self.assertEqual(result, {'test': 1, 'Sentiment': 0.0, 'Code Blocks': 0})
```

Comment:
I think it's important to also have test case that return non 0 values for sentiment and code blocks

Reply:
Done!!